src/main/webapp/script.js

```diff
@@ -214,10 +214,37 @@ function fetchAuth() {
  * Fetches and displays information related to mentor evidence.
  */
 function fetchMentorApproval() {
-  const mentor_id = (new URL(document.location)).searchParams.get('id');
-  const mentorApprovalUrl = '/mentor-approval?id=' + mentor_id.toString();
+  const mentorId = (new URL(document.location)).searchParams.get('id');
+  if (mentorId === null) {
+    window.location.replace('/index.html')
+  }
+  const mentorApprovalUrl = '/mentor-approval?id=' + mentorId;
   fetch(mentorApprovalUrl).then(response => response.json()).then(approval => {
-    if (approval.isApprover) {
+    if (approval.userId == mentorId && approval.isApproved) {
+      // If mentor has been approved, show corresponding message.
+      createApprovalMessage(1);
```

Comment:
for this and the rest of the statements, instead of using 1, 2, etc. you could use constants, something like MENTOR_IS_APPROVED or something

Reply:
Do I add global constants at top of file or how do you recommend doing this?

Reply:
mhmmmm maybe doing a separate file like we did in SqlConstants might be the way to go, but I'll leave it up to you